src/feeds/sources/goprices.ts

```diff
@@ -0,0 +1,85 @@
+import { FeedSource, SolinkSubmitterConfig } from '../../config'
+import { log } from '../../log'
+import { IPrice, PriceFeed } from '../PriceFeed'
+import axios from 'axios'
+import BigNumber from 'bignumber.js'
+
+export class GOPrices extends PriceFeed {
+  // go prices feed must have 6 decimals
+  public decimals = 6
+  protected log = log.child({ class: this.source })
+  protected baseurl = ''
+
+  constructor(
+    public source: FeedSource.GOPRICES_1 | FeedSource.GOPRICES_2,
```

Comment:
just use the `GOPRICES_1` as is.

each oracle would run their own go price feed